CHANGELOG.md

```diff
@@ -20,14 +20,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - `Terminate` request is not supported, and DAP configuration is such that it won't be requested by the client.
 
 - probe-rs-debugger: Improve handling of `restart` request. With support in DAP Client/VSCode for: (#1507)
+
   - `Restart` will now restart the debug session. Currently this is support for ARM targets only.
   - If a newer binary is available, and flashing enabled, then the new binary will be flashed before starting the new debug session.
 
+- probe-rs-debugger: Ensure VSCode will halt on early `main()`, irrespective of flashing config. (#1529)
```

Comment:
What is meant with an early `main()` here? It's not clear to me, maybe you you could describ it another way?

Reply:
The way it is described in the original problem report, is that breakpoints that are set before the `loop` in `main()`, can be skipped, because processing passed that point, before the VSCode `BreakPoints` request was received.

I updated the CHANGELOG with something more succinct, but it does have the reference to the original problem report :)